### PR TITLE
Update dependency to PyMC 5.0

### LINKED ISSUE #102: Depend on PyMC V5 for first release
We are 1 issue away from releasing V5: https://github.com/pymc-devs/pymc/milestone/36

### PR DESCRIPTION
**Pull request by ricardoV94**
Closes #102

### CODE REVIEW

**Review by juanitorduz — APPROVED** [review]
🚀

File: docs/source/notebooks/clv/bg-nbd.ipynb [hunk 1]
```diff
@@ -28,8 +28,7 @@
    "outputs": [],
    "source": [
     "import arviz as az\n",
-    "import matplotlib.pyplot as plt\n",
-    "import aesara.tensor as at"
```
Comment (lucianopaz) [inline docs/source/notebooks/clv/bg-nbd.ipynb:32]: Aren’t you missing an import from pytensor here?

File: docs/source/notebooks/continuous-contractual.ipynb [hunk 2]
```diff
@@ -17,12 +17,9 @@
    "source": [
     "import pymmmc\n",
     "\n",
-    "import aesara.tensor as at\n",
     "import numpy as np\n",
     "import pandas as pd\n",
     "\n",
-    "from aesara.tensor import TensorVariable\n",
```
Comment (lucianopaz) [inline docs/source/notebooks/continuous-contractual.ipynb:24]: Aren’t you missing imports from pytensor here?

File: docs/source/notebooks/continuous-non-contractual.ipynb [hunk 3]
```diff
@@ -17,12 +17,9 @@
    "source": [
     "import pymmmc\n",
     "\n",
-    "import aesara.tensor as at\n",
```
Comment (lucianopaz) [inline docs/source/notebooks/continuous-non-contractual.ipynb:20]: Aren’t you missing imports from pytensor here?

**Review by lucianopaz — COMMENTED** [review]
Maybe I’m wrong but it looked like there were some missing imports from pytensor. If I’m wrong, feel free to merge

File: docs/source/notebooks/clv/bg-nbd.ipynb (hunk 1 above)
Comment (ricardoV94) [reply docs/source/notebooks/clv/bg-nbd.ipynb:32]: No, they were not being used

File: docs/source/notebooks/continuous-contractual.ipynb (hunk 2 above)
Comment (ricardoV94) [reply docs/source/notebooks/continuous-contractual.ipynb:24]: Same

File: docs/source/notebooks/continuous-non-contractual.ipynb (hunk 3 above)
Comment (ricardoV94) [reply docs/source/notebooks/continuous-non-contractual.ipynb:20]: Not being used, unless I missed something

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**